rialto_airflow/dags/harvest.py

```diff
@@ -226,6 +243,8 @@ def upload_data_quality_files(snapshot):
 
     snapshot = load_authors(snapshot)
 
+    pubmed_harvest(snapshot)
```

Comment:
This is going to change once #387 is merged. I think as this stands since the return value of calling `pubmed_harvest()` isn't assigned to a variable that is a dependency for the later steps in the workflow those later steps could potentially run before this has completed?